diff --git a/es/form/components/FormItem/index.js b/es/form/components/FormItem/index.js
index 7d4771f91a7c65c91c47fa9d2a9198435c27dda8..97270c76cda63d8bd6ab1711b568788eb4cf50e0 100644
--- a/es/form/components/FormItem/index.js
+++ b/es/form/components/FormItem/index.js
@@ -196,6 +196,10 @@ const ProFormItem = props => {
     lightProps,
     children: unusedChildren,
     fieldProps,
+    // pnpm patch，移除这些字段，避免奇怪的问题。
+    onBlur,
+    cacheForSwr,
+    proFieldKey,
     ...rest
   } = props;
   const formListField = useContext(FormListContext);
